src/main/java/mate/academy/spring/security/PasswordValidator.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.spring.security;
+
+import java.util.Objects;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import mate.academy.spring.dto.request.UserRequestDto;
+
+public class PasswordValidator implements ConstraintValidator<Password, UserRequestDto> {
+    @Override
+    public boolean isValid(UserRequestDto user,
+                           ConstraintValidatorContext constraintValidatorContext) {
+        return Objects.equals(user.getPassword(), user.getRepeatPassword())
+                && user.getPassword().length() >= 4 && !user.getPassword().isBlank();
```

Comment:
should be such order of checks:
1. check isBlank
2. check length
3. compare passwords